worker_test: TestUtilityRlimit fails occasionally

```
?   	github.com/sjtug/lug	[no test files]
ok  	github.com/sjtug/lug/config	0.002s
?   	github.com/sjtug/lug/exporter	[no test files]
ok  	github.com/sjtug/lug/helper	0.006s
ok  	github.com/sjtug/lug/manager	6.004s
--- FAIL: TestUtilityRlimit (0.33s)
        Error Trace:    worker_test.go:94
	Error:		Should be true
		
time="2018-02-02T23:50:12+08:00" level=info msg="start execution" worker=shell 
time="2018-02-02T23:50:12+08:00" level=info msg="Exporter initialized" 
time="2018-02-02T23:50:12+08:00" level=info msg=succeed worker=shell 
time="2018-02-02T23:50:12+08:00" level=info msg="Stderr: " worker=shell 
FAIL
FAIL	github.com/sjtug/lug/worker	1.340s
```

--- a/worker/worker_test.go
+++ b/worker/worker_test.go
@@ -72,7 +72,7 @@ func (i *limitReader) Read(p []byte) (int, error) {
 	}
 	i.cnt += len(p)
 	for i := 0; i < len(p); i++ {
-		p[i] = 0
+		p[i] = 5 // shouldn't use zero here, because sometimes pages filled with zero are not allocated
 	}
 	return len(p), nil
 }

